Makefile

```diff
@@ -7,7 +7,7 @@
 REGISTRY := ethyca
 IMAGE_TAG := $(shell git fetch --force --tags && git describe --tags --dirty --always)
 
-IMAGE_NAME := fidesops_webserver
+IMAGE_NAME := webserver
```

Comment:
This one needs to stay `webserver` rather than `fidesops_webserver` or commands that don't use `run_infrastructure.py` will fail.

Reply:
ah ok, should this be a code comment for posterity?

Reply:
Yes, I can do that.